packages/components/src/form/src/Form.css

```diff
@@ -33,6 +25,8 @@
     margin-bottom: var(--o-ui-sp-3);
 }
 
-.o-ui-fieldset .o-ui-form-field:last-child {
-    margin-bottom: 0;
+.o-ui-fieldset-fields {
+    display: grid;
+    width: 100%;
+    gap: var(--o-ui-sp-5);
```

Comment:
I don't think the `width: 100%` is necessary and I am not sure it should be specified by default since Orbit usually a component to explicit receive a `fluid` property to display something as fluid.

Reply:
I'll dig down the reason it's been added and follow through